src/DatabaseExporter.php

```diff
@@ -302,7 +302,11 @@ protected function buildXmlData()
 				{
 					if (!in_array($key, $colblob))
 					{
-						$buffer[] = '    <field name="' . $key . '">' . htmlspecialchars($value, ENT_COMPAT, 'UTF-8') . '</field>';
+						if (is_null($value)) {
+							$buffer[] = '    <field name="' . $key . '" value_is_null></field>';
```

Comment:
```suggestion
							$buffer[] = '    <field name="' . $key . '">value_is_null</field>';
```
From reading the code I would expect the `value_is_null` being the value without a leading space and not a part of the field tag just after name.

Reply:
But I might be wrong.

Reply:
Reading deeper, I think I'm wrong.

Reply:
Yes, you are wrong. I definitely don't want to introduce some magic strings as values, which at some weird point will create yet another bug report. This is simply an empty attribute in the XML, nothing more.

Reply:
Yes, I got that a few seconds after I had written my first comment. Unfortunately it seems I can't mark that as resolved here.

Reply:
I already marked it as resolved.